certificate/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
In itself it is, but it could expose future weaknesses if the cert/passphrase is mishandled